node/node.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/mitchellh/mapstructure"
 	"github.com/sourcegraph/conc"
+	_ "go.uber.org/automaxprocs"
```

Comment:
I think this should be in `cmd/juno/juno.go`

Reply:
I don't think it matters, but will move it there if you think it makes more sense there.